xml/admin_gui_dashboard.xml

```diff
@@ -645,7 +645,7 @@
     <title>Purging Custom Roles</title>
     <para>
      If you create custom user roles and intend to remove the &ceph; cluster
-     with the <command>ceph.purge</command> runner later on, you need to purge
+     with the <command>ceph.purge</command> command later on, you need to purge
```

Comment:
`ceph.purge` itself is not a command, it's a `salt` runner or subcommand if you want. i would inclide the long version that can be actually run from the command line,which is `salt-run state.orch ceph.purge`
```suggestion
     with the <command>salt-run state.orch ceph.purge</command> command later on, you need to purge
```